mavis/python/mavis_tacplus_vault.py

```
@@ -0,0 +1,539 @@
+	try:
+		client = redis.Redis.from_url(
+			REDIS_URL, decode_responses=True, socket_timeout=2
+		)
+		client.ping()
+		_redis_client = client
+		_redis_fail_count = 0
+		print(
+			"mavis_tacplus_vault: connected to Redis at " + REDIS_URL,
+			file=sys.stderr,
+		)
+		return _redis_client
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)